app/src/main/java/com/hiddenramblings/tagmo/BluetoothLeService.java

```diff
@@ -129,7 +155,7 @@ private void getCharacteristicValue(BluetoothGattCharacteristic characteristic)
                         response = new StringBuilder();
                     }
                 } else if (progress.startsWith("tag.getList()")) {
-                    if (progress.endsWith(">")) {
+                    if (progress.endsWith(">") || progress.endsWith("\n")) {
```

Comment:
The `trim` should keep this from being a scenario for `progress` but it shouldn't hurt to leave it anyway.

Reply:
Good point. I will do some more testing and look to remove it.